src/api/make.js

```diff
@@ -37,6 +38,8 @@ export default async (providedOptions = {}) => {
     arch: electronHostArch(),
     platform: process.platform,
   }, providedOptions);
+
+  const outDir = providedOptions.outDir || `${dir}/out`;
```

Comment:
This won't / might not work on windows, please change to `path.resolve(dir, 'out')`